tests/keras_tests/exporter_tests/tflite_int8/test_models_runner.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import unittest
+
+from tests.keras_tests.exporter_tests.tflite_int8.networks.conv2d_test import TestConv2DExporter
+from tests.keras_tests.exporter_tests.tflite_int8.networks.dense_test import TestDenseExporter
+from tests.keras_tests.exporter_tests.tflite_int8.networks.depthwiseconv2d_test import TestDepthwiseConv2DExporter
+from tests.keras_tests.exporter_tests.tflite_int8.networks.mobilenetv2_test import TestMBV2Exporter
+
+
+
+class TFLiteINT8ExporterNetworksTest(unittest.TestCase):
```

Comment:
Why is the file called "models_runner"? Most of the tests are layers tests.
Anyway, I think you should rethink the structure of the exporter tests package, if there should be other exporter tests that follow these, then where they should be implemented? should there really be a runner for each type of exporter or maybe one runner for all? (I think the latter but we can talk about it).